src/AssetsPage/statusMap.json

```diff
@@ -0,0 +1,5 @@
+{
+  "504": "Studio is not running",
+  "404": "You are not logged in to Studio",
+  "403": "You are logged in to Studio, but your account doesn't have the correct permissions"
```

Comment:
Wouldn't hurt to define 500 here too so we know we broke API code when we are developing on the assets page later.